### DISCUSSION TITLE
dock 触发的位置太高

### DISCUSSION THREAD

**Discussion by wangling12** [linuxdeepin/developer-center#3233] [Features Request & Ideas | 特性请求 & 头脑风暴]
![截图录屏_选择区域_20210708141648](https://user-images.githubusercontent.com/24770600/124872019-54598080-dff7-11eb-8ad8-c3a5a088b342.gif)

例如放在右侧隐藏，那我就永远拖动不了滚动条了。
触发位置应该放在底边，而且指针以一定的速度撞上去或者指针停个0.5秒再触发比较合理。

**Comment by justforlxz** [comment]
任务栏也需要一定的区域，最低需要一像素去识别鼠标，但是窗口边缘恰好在最后一像素，我让开发调试一下吧，看能不能延长一些识别时间。

**Comment by wangling12** [comment]
> 任务栏也需要一定的区域，最低需要一像素去识别鼠标，但是窗口边缘恰好在最后一像素，我让开发调试一下吧，看能不能延长一些识别时间。

恩,最后一像素也行, 在一段时间无点击事件时触发. 现在这个实在太高了,连滚动条都点不到就触发了

**Comment by ssk-wh** [comment]
不建议这样设置，因为任务栏是隐藏模式，所以不会挤占桌面工作区域，导致任务栏区域和应用显示区域重叠，这样就肯定有冲突，设置1像素也不好，可能很难出发，我记得当前是15像素宽度的位置都是触发区域
可以调整成一直显示或者将任务栏设置在屏幕左边

**Comment by wangling12** [comment]
> 不建议这样设置，因为任务栏是隐藏模式，所以不会挤占桌面工作区域，导致任务栏区域和应用显示区域重叠，这样就肯定有冲突，设置1像素也不好，可能很难出发，我记得当前是15像素宽度的位置都是触发区域 可以调整成一直显示或者将任务栏设置在屏幕左边

dock的触发已经有一些比较成熟的方案了，鼠标在角落触发，鼠标速度大于一定的速度撞上去触发，鼠标在边框晃一晃触发等。

放在左边问题也依旧是问题，不会因为用的频率低了它就好了

![录屏_选择区域_20220120152016](https://user-images.githubusercontent.com/24770600/150291791-6bb8fc58-74d1-44e4-9b63-5e72ba4c4a83.gif)

至于改成一直显示，解决不了bug就解决功能是吧😂

**Comment by BLumia** [comment]
这个感觉更接近需求建议，我标成需求然后看看能怎样处理吧。cc @meiyixiang

**Comment by meiyixiang** [comment]
这是一个已知的问题，反馈了很久这需要整个系统去提升极致的交互体验。

**Comment by wangling12** [comment]
三年过去了,虽然旧的没有修,但我们可以来点新问题啊 :rofl: 
还是dde-dock, 最近发现放在底部的话,也会影响滚动条. 而且在`一直显示`的情况下,dde-dock似乎覆盖了一部分当前应用
录屏的时候很明显:移动到那底部附近鼠标就直接消失了,那块的鼠标左右键对上方浏览器及dde-dock都无效
这让浏览器本就不宽的滚动条更加点不中了
而在非最大化的情况下, 浏览器边缘可以正常点击
![录屏_选择区域_20240515164134](https://github.com/linuxdeepin/developer-center/assets/24770600/5ff83e4e-5dd1-44e9-8578-775f1f0407eb)

**Reply by BLumia** [reply]
你这也不是最新版本呀，最新版本（v23 rc）已经是 dde-shell 了。
